### PR TITLE
The correct registration of binary script for composer

### PR DESCRIPTION
**Pull request by hason**

### CODE REVIEW

File: bin/phpcrsh [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
-
-if (file_exists($file)) {
-    $autoload = require_once $file;
-} else {
-    echo 'Cannot find the vendor directory, have you executed composer install?';
-    exit(1);
+if (!class_exists('\PHPCR\Shell\PhpcrShell')) {
+    if (is_file(__DIR__.'/../vendor/autoload.php')) {
+        require __DIR__.'/../vendor/autoload.php';
+    } elseif (is_file(__DIR__.'/../../../autoload.php')) {
```
Comment (dbu) [inline bin/phpcrsh]: this is the case of binaries installed in `vendor/.composer/bin` right?

Comment (hason) [inline bin/phpcrsh]: Yes. This is case of symlink to binary from ``COMPOSER_BIN_DIR (default vendor/composer/bin)``  to ``vendor/phpcr/phpcr-shell/bin``.

**Comment by dantleech** [comment]
The build was failing ofr some random reason. Restarted now, if it is green feel free to merge if I havn't already done so (bad internet connectivity) @dbu

**Comment by dbu** [comment]
i can't see how the build failure would be related to the changes here. i restarted the last successful build on master, just to be sure: https://travis-ci.org/phpcr/phpcr-shell/builds/65419628

but i think its more that meanwhile there are too many symfony versions available and thus composer has too many options.

**Comment by dbu** [comment]
master now fails as well, so this really is not related. thanks for the fix!

created #170 about the build failures.

**Pull request merged**